layers/+readers/djvu/README.org

```diff
@@ -0,0 +1,87 @@
+#+TITLE: djvu layer
+#+TAGS: layer|reader
+
+# The maximum height of the logo should be 200 pixels.
+[[img/djvu-logo.svg]]
+
+# TOC links should be GitHub style anchors.
+* Table of Contents                                        :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+  - [[#prerequisites][Prerequisites]]
+  - [[#dotfile][Dotfile]]
+- [[#key-bindings][Key bindings]]
+
+* Description
+This layer adds support for reading djvu files with spacemacs.
+
+Because of some difficulties, the layer does not lazy load the =djvu.el= and
+=djvu3.el= packages.
+
+** Features:
+This layer provides a full featured djvu editor by implementing the [[https://github.com/dalanicolai/djvu2.el][djvu.el
+package]] along with the [[https://github.com/dalanicolai/djvu3][djvu3]] extension (a newer alternative to [[https://github.com/dalanicolai/djvu2.el][djvu2.el]]).
+
+- flexible annotation editing and rendering, covering most (but not all) of
+  [[https://linux.die.net/man/1/djvused][djvused]] annotation definitions
+- fast navigation with imenu
+- fast search with djvu-occur
+- remember last view with djvu-restore
+- dark mode
+
+* Install
+** Prerequisites
+  The =djvu= package requires the =djvused= command (from [[http://djvu.sourceforge.net/][djvulibre]]) to be
+  available in =exec-path=.
+
+  Additionally, for =dark-mode= availability of the command =pnminvert= (from
+  [[http://netpbm.sourceforge.net/][netpbm]]) is required.
+  
+** Dotfile
+    After installing the dependencies, add this to your ~/.spacemacs.
+
+#+BEGIN_SRC emacs-lisp
+(setq-default dotspacemacs-configuration-layers '(djvu))
+#+END_SRC
+
+After that, syncronize your configuration with SPC f e R.
+
+* Key bindings
+
+*djvu-read-mode*
+| Key Binding | Description                                                                      |
+|-------------+----------------------------------------------------------------------------------|
+| ~i~         | djvu-image-toggle                                                                |
+| ~j~         | scroll up                                                                        |
+| ~k~         | scroll down                                                                      |
+| ~J~         | next page                                                                        |
```

Comment:
that's okay.
but can you also include `h/l` for horizontal scrolling in the table?

then if `H/L` are not occupied, could it be used for command like previous/next chapter/section or bookmarked page?

Reply:
The djvu package does not provide horizontal scrolling commands. But they are not very needed, because when zooming in, the package stops rendering images already before the page width reaches half the(my) screensize...